src/domain/users/entities/member.entity.ts

```diff
@@ -31,7 +32,7 @@ export const MemberSchema: z.ZodType<
 > = RowSchema.extend({
   user: z.lazy(() => UserSchema),
   space: z.lazy(() => SpaceSchema),
-  name: z.string(),
+  name: NameSchema,
```

Comment:
This is the other meaningful change in the PR: `Member` names were plain strings.